typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -6,6 +6,8 @@ export enum WarpRouteIds {
   ArbitrumNeutronTIA = 'TIA/arbitrum-neutron',
   EclipseSolanaSOL = 'SOL/eclipsemainnet-solanamainnet',
   EclipseSolanaWIF = 'WIF/eclipsemainnet-solanamainnet',
+  EclipseStrideSTTIA = 'STTIA/eclipse-stride',
```

Comment:
I think setting this value to stTIA/eclipse-stride should allow the checker to find the addresses in registry and solve your problem